modules/affile/affile-dest.c

```diff
@@ -97,7 +97,7 @@ struct _AFFileDestWriter
   time_t last_msg_stamp;
   time_t last_open_stamp;
   time_t time_reopen;
-  struct iv_timer reap_timer;
+  struct iv_timer retry_timer;
```

Comment:
My previous solution was using two timers but @bazsi is right and adding a redundant timer when the other one is not active is a waste per file destination. You're right about the multiple responsibility. Although, I think the responsibility of a timer is to be shot at the given time. The handlers split the functionality not the timers.